api/src/test/java/org/openmrs/ConceptNumericTest.java

```diff
@@ -122,6 +122,7 @@ public void shouldChangeConceptMapReferenceToParentConcept() throws Exception {
 	public void shouldSaveAConceptNumericWithAllowDecimalValue() throws Exception {
 		Concept c = Context.getConceptService().getConcept(22);
 		ConceptNumeric cn = new ConceptNumeric(c);
+		cn.addDescription(new ConceptDescription("some description", null));
```

Comment:
@wluyima Changed the locale to null and Updated ConceptSaveHandler,so that it sets a locale if any null locale is found